config/config.go

```diff
@@ -353,6 +353,14 @@ type (
 		RangeBloomFilterNumHash uint64 `yaml:"rangeBloomFilterNumHash"`
 	}
 
+	// Tracer is the config for tracer
+	Tracer struct {
+		// EndPoint the jaeger endpoint
+		EndPoint string `yaml:"endpoint"`
+		// InstanceID MUST be unique for each instance of the same
+		InstanceID string `yaml:"instanceID"`
```

Comment:
does InstanceID need to be in config? since it is unique for each instance, it should be assigned to a random number in the `New()`?

Reply:
The instanceID should be easy to quickly locate the container/machine, like `us-aws-node1` , and make new random number (restart server) will break tracing.